CONTRIBUTING/RELEASING.md

```diff
@@ -381,7 +381,50 @@ Before you start you should make sure that your working tree is clean and the re
 
 ## Canary Releases
```

Comment:
@JReinhold spotted a couple of items that need a bit of attention. Do you mind if I take a stab at this and polish the documentation and update it here?

Let me know and we'll go from there.

Reply:
@JReinhold just pushed a commit with some wording tweaks based on the latest iteration. Let me know if you're ok with them and we'll go from there.

Reply:
Looking good!